Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +9,24 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0)
```

Comment:
Didn't teach yet, but you can use array?.Length > 0. It doesn both the null check and the comparison